pkg/ingress/controller/controller.go

```diff
@@ -137,7 +137,7 @@ func NewController(cfg *config.Config) (*Controller, error) {
 	if cfg.Kubernetes.IngressVersion == config.IngressNetworkingV1 {
 		ingressInformer = kube.CoreSharedInformerFactory.Networking().V1().Ingresses().Informer()
 	} else {
-		ingressInformer = kube.CoreSharedInformerFactory.Extensions().V1beta1().Ingresses().Informer()
+		ingressInformer = kube.CoreSharedInformerFactory.Networking().V1beta1().Ingresses().Informer()
```

Comment:
Need to declare the minimum version of k8s supported by Ingress: 1.14 .
Please update the README by the way.

Reply:
done.